### ISSUE TITLE
Prevent infinite scrolling when the table view is not full of entries

### ISSUE THREAD

**Issue by arturgrigor** [samvermette/SVPullToRefresh#101]
There should be a way to prevent infinite scrolling when the table view is not full of entries. You can edit the code from the demo and reproduce the issue: 

Change:
    for(int i=0; i<15; i++)
        [self.dataSource addObject:[NSDate dateWithTimeIntervalSinceNow:-(i_90)]];
with:
    for(int i=0; i<1; i++)
        [self.dataSource addObject:[NSDate dateWithTimeIntervalSinceNow:-(i_90)]];

Maybe you should add another property to the infinite scrolling, like allowedScrollingDirection, and do the logic inside - (void)scrollViewDidScroll:(CGPoint)contentOffset. That's just a thought.

**Comment by samvermette** [comment]
I agree that this is a problem, but I'm not sure infinite scrolling should automatically get disabled when the table view's content height is smaller than its frame's. I guess infinite scrolling only makes sense when the viewport is full of data, so this would probably be appropriate. Thanks for reporting this.
